forge-gui-mobile/src/forge/adventure/stage/GameHUD.java

```diff
@@ -230,19 +238,22 @@ public void draw() {
     }
 
     Texture miniMapTexture;
+    Texture miniMapToolTipTexture;
+    Pixmap miniMapToolTipPixmap;
     public void enter() {
         if(miniMapTexture!=null)
             miniMapTexture.dispose();
         miniMapTexture=new Texture(WorldSave.getCurrentSave().getWorld().getBiomeImage());
-
+        if(miniMapToolTipTexture!=null)
+            miniMapToolTipTexture.dispose();
+        if(miniMapToolTipPixmap!=null)
+            miniMapToolTipPixmap.dispose();
+        miniMapToolTipPixmap=new Pixmap((int) (miniMap.getWidth()*3), (int) (miniMap.getHeight()*3), Pixmap.Format.RGBA8888);
+        miniMapToolTipPixmap.drawPixmap(WorldSave.getCurrentSave().getWorld().getBiomeImage(),0,0,WorldSave.getCurrentSave().getWorld().getBiomeImage().getWidth(),WorldSave.getCurrentSave().getWorld().getBiomeImage().getHeight(),0,0,miniMapToolTipPixmap.getWidth(),miniMapToolTipPixmap.getHeight());
+        miniMapToolTipTexture=new Texture(miniMapToolTipPixmap);
         miniMap.setDrawable(new TextureRegionDrawable(miniMapTexture));
+        miniMapTooltip.setDrawable(new TextureRegionDrawable(miniMapToolTipTexture));
         avatar.setDrawable(new TextureRegionDrawable(Current.player().avatar()));
-        Deck deck = AdventurePlayer.current().getSelectedDeck();
```

Comment:
I added this as an Indicator to player that they have empty or incomplete  deck selected so they don't battle with deck full of wastes accidentally incase the selected deck is empty (via accident or for some unknown reason). Do you have an alternative replacement for this or any hints to the player?

Reply:
I thought I merged it, but as you can see the commits are quite a mess right now apperently it got removed again, I will check it later